lib/iris/tests/stock.py

```diff
@@ -46,12 +46,12 @@ def lat_lon_cube():
     """
     cube = Cube(np.arange(12, dtype=np.int32).reshape((3, 4)))
     cs = GeogCS(6371229)
-    coord = iris.coords.DimCoord(points=np.array([-1, 0, 1], dtype=np.int32),
+    coord = DimCoord(points=np.array([-1, 0, 1], dtype=np.int32),
```

Comment:
If you do this, you can just remove the imports:
```
import iris.coords
import iris.coords as icoords
```
(which as the same thing)  as they never get used